popup.css

```diff
@@ -0,0 +1,116 @@
+.model {
+  box-sizing: border-box;
+  display: none;
+  position: fixed;
+  top: 410px;
+  left: 50%;
+  transform: translate(-50%, -55%);
+  transition: 200ms ease-in;
+  border: 1px solid #dfe1e6;
+  border-radius: 20px;
+  background-color: #fff;
+  padding: 30px;
+  z-index: 10;
+  width: 500px;
+  max-width: 80%;
+  font-family: 'poppins', sans-serif;
+}
+
+.model .btn {
+  display: flex;
+  padding: 20px 0;
+  justify-content: space-between;
+}
+
+.model-container {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  width: 100%;
+}
+
+.model-headline {
+  display: flex;
+  justify-content: space-between;
+}
+
+.model-headline button {
+  font-size: 40px;
+  cursor: pointer;
+  border: none;
+  outline: none;
+  background: none;
+  color: #67798e;
+}
+
+.model-headline h2 {
+  padding: 10px;
+}
+
+.button-list {
+  padding: 0;
+  display: flex;
+  justify-content: flex-start;
+}
+
+.button-list li {
+  padding-right: 20px;
+}
+
+.mobile-banners-model {
+  display: none;
+}
+
+.dogWater {
+  text-decoration: none;
+  color: white;
+}
+
+.btn button {
+  padding: 10px;
+}
+
+#overlay {
+  position: fixed;
+  display: none;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: rgba(193, 199, 208, 0.678);
+  filter: blur(10px);
+  z-index: 1;
+}
+
+@media (min-width: 768px) {
+  .model {
+    margin: 0 auto;
+    width: 45.75rem;
+
+    /* height: 50%; */
+    font-size: 0.625rem;
+  }
+
+  .mobile-banners-model {
+    display: block;
+    height: 50vh;
+  }
+
+  .work-banners-model {
+    display: none;
+  }
+
+  .parlist {
+    display: flex;
+    flex-direction: row;
+    gap: 20px;
+  }
+
+  .lister {
+    font-size: 12px;
+  }
+
+  .model-container {
+    max-width: 60.75rem;
+  }
```

Comment:
Great job @MohamedCK & @PraisesPJMT 🤝

- [ ] Just like the previous reviewer pointed out, you may kindly fix the popup on both mobile and desktop designs. Your popups are both fixed to the bottom. You cold add `margin-botton` to your popups.
- Your mobile popup
![mobile popup](https://user-images.githubusercontent.com/63917725/180281878-f2ae148d-e2f0-4d7c-918e-39c11511fc7b.PNG)

- Your desktop popup
![desktop popup](https://user-images.githubusercontent.com/63917725/180282050-0b2e4ff0-78ae-4a46-9885-061211d84361.PNG)